sipi/images/0001/Cpl5d73kOLz-FclZg2VVf6r.info

```diff
@@ -0,0 +1,7 @@
+{
+   "originalFilename": "standoffTransformation.xsl",
+   "checksumOriginal": "42ab2039da04d27446abc686fe93800e95cacf0935932bc4b001feb2243f3129",
+   "internalFilename": "Cpl5d73kOLz-FclZg2VVf6r.xsl",
+   "originalInternalFilename": "Cpl5d73kOLz-FclZg2VVf6r.xsl.orig",
+   "checksumDerivative": "42ab2039da04d27446abc686fe93800e95cacf0935932bc4b001feb2243f3129"
+}
```

Comment:
No newline at the end of file.

Reply:
that's how SIPI generated it, so I'll leave it like this.